experiments/pod-delete/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM jfloff/alpine-python:2.7-slim
+FROM alpine:latest
 
-RUN pip install jinja2-cli
+LABEL maintainer="LitmusChaos"
 
-ARG KUBECTL_VERSION=1.17.0
-ADD https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl /usr/local/bin/kubectl
-RUN chmod +x /usr/local/bin/kubectl
+#Installing necessary ubuntu packages
+RUN apk update && apk add curl bash
+RUN apk add --no-cache libc6-compat
 
-COPY pod-delete.sh /
-COPY pod-delete-event.yaml /
+#Copying Necessary Files
+COPY ./build/_output ./experiments
```

Comment:
Why we are changing this Dockerfile?

Reply:
& If we are deprecating  `experiment/pod-delete`( the bash one) then we should remove this from here, Travis & Makefile